webpack.builtinblocks.config.js

```diff
@@ -46,11 +50,6 @@ module.exports = {
             './xmodule/js/src/xmodule.js',
             './xmodule/js/src/raw/edit/metadata-only.js'
         ],
-        LibraryContentBlockDisplay: './xmodule/js/src/xmodule.js',
```

Comment:
It's unused. If it were used, there'd be a matching `add_wepback_js_to_fragment` call, [like this one](https://github.com/openedx/edx-platform/blob/af69cbe6714f21d70d77a27ad75d00bdfee3911d/xmodule/library_content_block.py#L471). 

Instead of going through a webpack bundle, the block's [student_view just adds its JS the standard xblock way](https://github.com/openedx/edx-platform/blob/af69cbe6714f21d70d77a27ad75d00bdfee3911d/xmodule/library_content_block.py#L416).

I added these entrypoints in the first place (#32481), and it's possible that I originally added this one erroneously.